src/static/qrcode.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en" class="TridactylOwnNamespace">
+    <title>QR Code display</title>
+    <head>
+        <meta charset="utf-8">
+        <title>&nbsp;</title>
+        <link rel="stylesheet" href="css/qrcode.css">
+        <link rel="stylesheet" href="css/content.css">
+        <link rel="stylesheet" href="css/hint.css">
+        <link rel="stylesheet" href="css/viewsource.css">
+    </head>
+    <body>
+      <div class="container">
+        <div id="qr-canvas">
+          <img>
+        </div>
+        <div id="error-display" class="hide error">
+        </div>
+        <div class="buttons" id="qr-buttons">
+          <a href="" download hidden></a>
+          <button id="download">Download</button>
+        </div>
```

Comment:
I'm not really keen on the button, I think it makes the page look a little cluttered. People can right click or use the save image hint mode.

Did you have a specific use-case in mind?

Reply:
addressed this below. Pls check commit #[a467747](https://github.com/tridactyl/tridactyl/pull/4728/commits/a467747057b0fb4e6e5b0af010c77f776574c93e)